core/src/main/java/com/linecorp/armeria/server/RoutePathType.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+/**
+ * The type of the path which was specified when a {@link Route} is created.
+ *
+ * @see RouteBuilder#path(String)
+ */
+public enum RoutePathType {
+
+    /**
+     * The exact path type. e.g, "/foo"
+     */
+    EXACT(true),
+
+    /**
+     * The prefix path type. e.g, "/", "/foo/"
+     */
+    PREFIX(true),
+
+    /**
+     * The path which contains path parameters. e.g, "/:", "/foo/:/bar/:"
+     */
+    PATH_PARAM(true),
```

Comment:
`PARAMETERIZED`? Please feel free to suggest a better name.

Reply:
I can't think of any better name. `PARAMETERIZED` is fine for me. Thanks!